src/Services/Masa.Auth.Service.Admin/Application/Subjects/Commands/UpdateUserCommandValidator.cs

```diff
@@ -7,12 +7,6 @@ public class UpdateUserCommandValidator : AbstractValidator<UpdateUserCommand>
 {
     public UpdateUserCommandValidator()
     {
-        RuleFor(command => command.User.DisplayName).Required().MaximumLength(50);
-        RuleFor(command => command.User.Name).ChineseLetterNumber().MaximumLength(20);
-        RuleFor(command => command.User.PhoneNumber).Phone();
-        RuleFor(command => command.User.Email).Email();
-        RuleFor(command => command.User.IdCard).IdCard();
-        RuleFor(command => command.User.CompanyName).ChineseLetterNumber().MaximumLength(50);
-        RuleFor(command => command.User.Position).ChineseLetterNumber().MaximumLength(20);
+        RuleFor(command => command.User).SetValidator(new UpdateUserValidator());
```

Comment:
Form validation rules for client and server are different,for example, a avatar is not required on the server side.